coffea/ml_tools/torch_wrapper.py

```diff
@@ -37,7 +37,11 @@ def __init__(self, torch_model: torch.nn.Module):
 
         # Reference to the original pytorch model, loading in the state, and
         # set to evaluation mode
-        self.orig_model = torch_model
+        self._dask_model = dask.delayed(
```

Comment:
@yimuchen so you need to be a bit careful here! If you persist the model then when you ask for it later it has to get it from wherever it was put! This might be on distributed compute and getting it would be slow! So you need a local copy.

What I did in lookup_base was to keep both weakref (for local evaluation) and a dask handle, so it's efficient on both sides of the fence.

Reply:
You can do silly things like:
```
self._model = weakref.ref(torch_model)
self._dask_model = dask.delayed(self.model(),...)
```

Reply:
Right, I'm still in the processes of testing this new wrapping. Is there a nice way to test for potential issues when scaling up?

Reply:
Got the first working version, now attempting to add additional test routines for the dask.persist fall back method.